doc/tutorials/mediasdk-tutorials-readme.md

```diff
@@ -0,0 +1,285 @@
+
+Notice revision #20110804
+
+<div style="page-break-before:always" />
+
+# Table of contents
+- [Introduction](#introduction)
+  - [Exploring Media SDK via a simplified set of samples](#exploring-Media-SDK-via-a-simplified-set-of-samples)
+  - [Windows specific notes](#windows-specific-notes)
+  - [How to obtain input content](#how-to-obtain-input-content)
+  - [Tutorials Samples Index](#tutorials-samples-index)
+- [Tutorial Section 1](#tutorial-section-1)
+  - [Hello World](#Hello-World)
+- [Tutorial Section 2: Decode](#tutorial-section-2-decode)
+  - [simple_2_decode](#simple_2_decode)
+  - [simple_2_decode_vmem](#simple_2_decode_vmem)
+- [Tutorial Section 3: Encode](#tutorial-section-3-encode)
+  - [simple_3_encode](#simple_3_encode)
+  - [simple_3_encode_vmem](#simple_3_encode_vmem)
+  - [simple_3_encode_vmem_async](#simple_3_encode_vmem_async)
+- [Tutorial Section 4: VPP](#tutorial-section-4-vpp)
+  - [simple_4_vpp_resize_denoise](#simple_4_vpp_resize_denoise)
+  - [simple_4_vpp_resize_denoise_vmem](#simple_4_vpp_resize_denoise_vmem)
+- [Tutorial Section 5: Transcode](#tutorial-section-5-transcode)
+  - [simple_5_transcode](#simple_5_transcode)
+  - [simple_5_transcode_opaque](#simple_5_transcode_opaque)
+  - [simple_5_transcode_opaque_async](#simple_5_transcode_opaque_async)
+  - [simple_5_transcode_opaque_async_vppresize](#simple_5_transcode_opaque_async_vppresize)
+  - [simple_5_transcode_vmem](#simple_5_transcode_vmem)
+- [Tutorial Section 6: Advanced Media SDK](#tutorial-section-6-advanced-media-sdk)
+  - [How to create low latency pipelines and how to benchmark latency](#how-to-create-low-latency-pipelines-and-how-to-benchmark-latency)
+    - [simple_6_encode_vmem_lowlatency](#simple_6_encode_vmem_lowlatency)
+    - [simple_6_transcode_opaque_lowlatency](#simple_6_transcode_opaque_lowlatency)
+  - [How to use VPP for pre- or post-processing purposes](#how-to-use-vpp-for-pre--or-post-processing-purposes)
+  - [simple_6_decode_vpp_postproc](#simple_6_decode_vpp_postproc)
+  - [simple_6_encode_vmem_vpp_preproc](#simple_6_encode_vmem_vpp_preproc)
+<div style="page-break-before:always" />
+
+# Introduction
+## Exploring Media SDK via a simplified set of samples
+The Intel® Media Software Development Kit (Intel® Media SDK) gives developers access to specialized hardware acceleration capabilities for video decoding, encoding, and pre/postprocessing (VPP).
+
+Intel® Media SDK helps developers rapidly write software that accesses hardware acceleration for video codecs with automatic fallback on software if hardware acceleration is not available. Intel® Media SDK is available for downloading here: http://www.intel.com/software/mediasdk/.
```

Comment:
We need to correct/review where mediasdk can be obtained from... Or maybe just skip that,,,